osu.Game/Screens/Play/HUD/UnstableRateCounter.cs

```diff
@@ -3,48 +3,29 @@
 
 using System;
 using osu.Framework.Allocation;
-using osu.Framework.Bindables;
-using osu.Framework.Extensions.ObjectExtensions;
-using osu.Framework.Graphics;
-using osu.Framework.Graphics.Containers;
-using osu.Framework.Graphics.Sprites;
-using osu.Framework.Localisation;
-using osu.Game.Graphics;
-using osu.Game.Graphics.Sprites;
 using osu.Game.Graphics.UserInterface;
 using osu.Game.Rulesets.Judgements;
 using osu.Game.Rulesets.Scoring;
-using osu.Game.Skinning;
-using osuTK;
 
 namespace osu.Game.Screens.Play.HUD
 {
-    public partial class UnstableRateCounter : RollingCounter<int>, ISerialisableDrawable
+    public abstract partial class UnstableRateCounter : RollingCounter<int>
     {
         public bool UsesFixedAnchor { get; set; }
 
         protected override double RollingDuration => 375;
 
-        private const float alpha_when_invalid = 0.3f;
-        private readonly Bindable<bool> valid = new Bindable<bool>();
-
         private HitEventExtensions.UnstableRateCalculationResult? unstableRateResult;
 
         [Resolved]
         private ScoreProcessor scoreProcessor { get; set; } = null!;
 
-        public UnstableRateCounter()
+        protected UnstableRateCounter()
         {
             Current.Value = 0;
         }
 
-        [BackgroundDependencyLoader]
-        private void load(OsuColour colours)
-        {
-            Colour = colours.BlueLighter;
-            valid.BindValueChanged(e =>
-                DrawableCount.FadeTo(e.NewValue ? 1 : alpha_when_invalid, 1000, Easing.OutQuint));
-        }
+        public virtual bool IsValid { get; set; }
```

Comment:
not sure why this was changed *away* from being a bindable, only to then have inheritors of the class *override the setter* and essentially make it a bindable again. just make it a bindable again please